### DISCUSSION TITLE
using swiftagram can lead to apple ban?

### DISCUSSION THREAD

**Discussion by slehydershokworks** [sbertix/Swiftagram#313] [Q&A]
I am using the app to get the last post of the logged in user.
I have doubts if this can lead to apple blocking my store application.

**Comment by sbertix** [comment]
Hey @slehydershokworks. 
Yes, Apple may reject apps using **Swiftagram**, and they're usually reviewed case-by-case. 
As a reference, [all unathorized third-party APIs, cannot be considered App Store safe](https://9to5mac.com/2020/08/27/apple-rejects-watch-for-tesla-app-as-it-starts-requiring-written-consent-for-third-party-api-use/).
